index.css

```diff
@@ -462,11 +442,8 @@ header li a:hover {
 }
 
 .page3 .container .nested {
```

Comment:
- [OPTIANAL ] As required by the previous review, you can decrease the font size of the social media icons as well as some margin and padding, You must replicate the design as much as possible (e.g., font, colors, images, tests, margins) Make sure you fix that before merging this PR
> 
> Your design:
> 
> ![05](https://user-images.githubusercontent.com/57562869/149399063-9ab03dd4-687e-48c1-85b7-270f0689573b.png)
> 
> Figma design:
> 
> ![052](https://user-images.githubusercontent.com/57562869/149399106-80f8e396-a59d-4866-a744-ea881255437e.png)
>